src/limbo/tools/sys.hpp

```diff
@@ -51,6 +51,12 @@
 #include <string>
 #include <unistd.h>
 
+#if defined(__MINGW32__) || defined(_MSC_VER)
+#include <winsock2.h>
+#else
+#include <unistd.h>
```

Comment:
You are including `unistd.h` two times.. See line 52 as well.

Reply:
Thanks for the hint, I forgot to remove the first include of `unistd.h` when moving the code to the fork. It should be fixed now in the last commit. I checked again that it still compiles with MinGW.